internal/backends/mcu/stm32h735g.rs

```diff
@@ -99,6 +112,82 @@ pub fn init() {
         dp.OCTOSPI2.octospi_hyperbus_unchecked(config, &ccdr.clocks, ccdr.peripheral.OCTOSPI2);
     let hyperram_ptr: *mut u32 = hyperram.init();
 
+    let _ncs = gpiog.pg6.into_alternate::<10>().speed(High).internal_pull_up(true);
+    let _clk = gpiof.pf10.into_alternate::<9>().speed(High).internal_pull_up(true);
+    let _dqs = gpiob.pb2.into_alternate::<10>().speed(High).internal_pull_up(true);
+    let _io0 = gpiod.pd11.into_alternate::<9>().speed(High).internal_pull_up(true);
+    let _io1 = gpiod.pd12.into_alternate::<9>().speed(High).internal_pull_up(true);
+    let _io2 = gpioe.pe2.into_alternate::<9>().speed(High).internal_pull_up(true);
+    let _io3 = gpiod.pd13.into_alternate::<9>().speed(High).internal_pull_up(true);
+    let _io4 = gpiod.pd4.into_alternate::<10>().speed(High).internal_pull_up(true);
+    let _io5 = gpiod.pd5.into_alternate::<10>().speed(High).internal_pull_up(true);
+    let _io6 = gpiog.pg9.into_alternate::<9>().speed(High).internal_pull_up(true);
+    let _io7 = gpiod.pd7.into_alternate::<10>().speed(High).internal_pull_up(true);
+
+    use stm32h7xx_hal::xspi::*;
+    use OctospiWord as XW;
+
+    let mut octospi =
+        dp.OCTOSPI1.octospi_unchecked(12.MHz(), &ccdr.clocks, ccdr.peripheral.OCTOSPI1);
+
+    octospi.configure_mode(OctospiMode::OneBit).unwrap();
+
+    // RDID Read Identification. Abuses address as instruction phase, but that
+    // works in SPI mode.
+    let mut read: [u8; 3] = [0; 3];
+    octospi.read(0x9F, &mut read).unwrap();
+
+    // Switch Macronix MX25LM51245GXDI00 to SDR OPI
+    // Set WREN bit
+    octospi.write_extended(XW::U8(0x06), XW::None, XW::None, &[]).unwrap();
+    // Write Configuration Register 2
+    octospi.write_extended(XW::U8(0x72), XW::U32(0), XW::None, &[1]).unwrap();
+    // Change bus mode
+    octospi.configure_mode(OctospiMode::EightBit).unwrap();
+
+    const MX25LM51245G_OCTA_READ_CFG_REG2_CMD: u16 = 0x718E;
+    const MX25LM51245G_CR2_REG1_ADDR: u32 = 0x00000000;
+    const MX25LM51245G_OCTA_READ_CMD: u16 = 0xEC13;
+
+    // check the config register
+    let mut read: [u8; 1] = [0];
+    octospi
+        .read_extended(
+            XW::U16(MX25LM51245G_OCTA_READ_CFG_REG2_CMD),
+            XW::U32(MX25LM51245G_CR2_REG1_ADDR),
+            XW::None,
+            5,
+            &mut read,
+        )
+        .unwrap();
+    assert_eq!(read[0], 1);
```

Comment:
I’m curious, what is this checking?:)

Reply:
that the write to the config register worked